registration/forms.py

```diff
@@ -86,14 +86,15 @@ class Textarea(InputStyleMixin, forms.widgets.Textarea):
 
 # Forms #######################################################################
 
-class LoginForm(NgFormValidationMixin, AuthenticationForm, metaclass=NgDeclarativeFieldsMetaclass):
+class LoginForm(AuthenticationForm):
     """
     Allows users to login with username/email and password.
     """
     username = forms.CharField(
         label='Your email or username',
         help_text='You can enter either your username or your email to login.',
         widget=TextInput,
+        max_length=150
```

Comment:
This looks like a magic number but I have just tried to add the client-side validation added by djng earlier, should we remove it?

Reply:
I'm guessing this comes from django contrib's auth `User` models, so I think we should leave it in.